src/Components/ImagesTable/ImageLink.js

```diff
@@ -1,126 +1,49 @@
-import React from 'react';
+import React, { Suspense } from 'react';
 import PropTypes from 'prop-types';
-
-import {
-  Button,
-  Popover,
-  Text,
-  TextContent,
-  TextVariants,
-} from '@patternfly/react-core';
-import { DownloadIcon, ExternalLinkAltIcon } from '@patternfly/react-icons';
+import { Button } from '@patternfly/react-core';
+import { useLoadModule, useScalprum } from '@scalprum/react-core';
+import ImageLinkDirect from './ImageLinkDirect';
 
 const ImageLink = (props) => {
-  const fileExtensions = {
-    vsphere: '.vmdk',
-    'guest-image': '.qcow2',
-    'image-installer': '.iso',
-  };
+  const scalprum = useScalprum();
+  const hasProvisionig = scalprum.initialized && scalprum.config?.provisioning;
+  if (hasProvisionig && props.imageType === 'ami') {
```

Comment:
first gating makes sure we won't error out on `useLoadModule` which is only safe to use, if the loaded app is in the environment.

Reply:
lgtm then :)